### ISSUE TITLE
Implement a Photo Viewer

### ISSUE THREAD

**Issue by huyouare** [huyouare/Duke-Study-Group#1]

**Comment by JustinLovesCompsci** [comment]
Where should the photo viewer be?

**Comment by huyouare** [comment]
It should be presented whenever a user touches inside an image presented in the ChatViewController. We should just find something on GitHub.

**Comment by huyouare** [comment]
@JustinLovesCompsci Sorry if I merged this too soon.

If you get a chance to work on it later, the transition looks a bit weird and maybe there should be a button to exit. Optionally get videos working.

There's a bug with clicking on Settings when in photo viewer, since "groups[0]" is hard-coded in.
![screen shot 2015-04-12 at 11 56 31 pm](https://cloud.githubusercontent.com/assets/3528340/7109633/97f42724-e16f-11e4-99c1-110222157ab9.png)
![screen shot 2015-04-12 at 11 56 40 pm](https://cloud.githubusercontent.com/assets/3528340/7109634/97f4e0a6-e16f-11e4-9112-cc85fc5b7943.png)

**Comment by JustinLovesCompsci** [comment]
You are right. I realized the transition is a bit weird. I will use an open source image viewer instead. On this now.

Regarding the bug, did you open settings in Chat view (with no group)? Since pure chat view will be removed #27  (based on what we discussed during the last meeting), this will not happen when we release the app. This bug happens because the chat is not associated with any group in the database table, which cannot happen once we remove the message tab view
